src/atoms/button/button-styles.js

```diff
@@ -7,7 +7,7 @@ import { outline, raised, nostyle } from 'styles';
 
 const hoverStyles = css`
 	&:hover:not(:active):not(:focus) {
-		background: ${props => darken(0.2, themeGet(`colors.${props.type}.main`)(props))};
+		background: ${props => darken(0.2, themeGet(`colors.${props.type}.main`)(props))}};
```

Comment:
Im not sure what this new `}` closes

```suggestion
		background: ${props => darken(0.2, themeGet(`colors.${props.type}.main`)(props))};
```